packages/ui/src/components/DataMapper/debug/README.md

```diff
@@ -0,0 +1,7 @@
+### How to use
+
+#### In this directory
+`yarn vite .`
```

Comment:
not working wih same error than https://github.com/KaotoIO/kaoto/pull/1700/files#r1876206691

Reply:
@igarashitm , here we need to refer to the preparation instructions in the README.md